### DISCUSSION TITLE
Scope down or change root of the tree

### DISCUSSION THREAD

**Discussion by lawrence-laz** [nvim-tree/nvim-tree.lua#1252] [Q&A] [answered]
Pressing `-` calls `dir_up` and changes root directory to the one above. This is good for when you need something above the current project tree. However once you are done with those files, the tree now has a different root and is a bit harder to work with, as it might have many unnecessary items displayed.

After I am done working in the upper directory I would like to "scope" the tree back into some other child directory. Basically I want to change the root directory to some other.

I went through the `:help NvimTree` but couldn't really find anything like "dir_down", or "change_root", etc.

**Comment by alex-courtis** [comment]
Here's a custom action I used to use: https://github.com/alex-courtis/arch/blob/c91322f51c41c2ace04614eff3a37f13e5f22a24/config/nvim/lua/nvt.lua#L9

`.` takes you to vim's global cwd

`actions.change_dir.global = false` is needed.

**Comment by alex-courtis** [comment, marked as answer]
Alternative:

If you want to change to a directory other than vim's current cwd you can try `actions.change_dir.global = true`.

That allows you to change the root to a directory of your choosing using the cd action: `<C-]>`

**Reply by lawrence-laz** [reply]
<kbd><C-]></kbd> was what I was looking for!

Looking at the docs again, now that I know what I'm looking for I found this line:
```
- typing <C-]> will cd in the directory under the cursor
```
I guess it was a bit hard to find because a bit different wording is used in the related command:
```
- type - to navigate up one directory
```
